sessions/006/README.md

```diff
@@ -551,3 +551,328 @@ pvc-0bd38196-2f5b-4a05-8917-b43bd2dde50b   20Gi       RWO            Delete
+
+```sh
+$ kubectl delete pv $(kubectl get pv | grep Available | awk '{print $1}')
+persistentvolume "pvc-8f3ddfba-e73a-4469-92e1-c8abc62006cc" deleted
+persistentvolume "pvc-90f07fde-7482-4764-b013-a04aac4e41fb" deleted
+persistentvolume "pvc-f6b43572-30ff-4423-a2eb-50d6733c55ec" deleted
+
+$ kubectl patch pv $(kubectl get pv | grep "$CLUSTER_NAME-zookeeper" | awk '{print $1}') --type merge -p '
+    spec:
+      persistentVolumeReclaimPolicy: Delete'
+persistentvolume/pvc-62dc40e2-8732-4563-bd9c-97a480f35dbf patched
+persistentvolume/pvc-d33f37a9-de7e-4b6a-8ab1-e0522571ebb1 patched
+persistentvolume/pvc-e56566de-8b03-4505-8872-f12603dea51f patched
+
+$ kubectl get pv | grep $CLUSTER_NAME-zookeeper
+pvc-62dc40e2-8732-4563-bd9c-97a480f35dbf   100Mi      RWO            Delete           Bound    test/data-my-cluster-zookeeper-2                                     ocs-external-storagecluster-ceph-rbd            6m28s
+pvc-d33f37a9-de7e-4b6a-8ab1-e0522571ebb1   100Mi      RWO            Delete           Bound    test/data-my-cluster-zookeeper-1                                     ocs-external-storagecluster-ceph-rbd            6m30s
+pvc-e56566de-8b03-4505-8872-f12603dea51f   100Mi      RWO            Delete           Bound    test/data-my-cluster-zookeeper-0                                     ocs-external-storagecluster-ceph-rbd            6m31s
+```
+
+<br/>
+
+---
+### Example: online ZooKeeper recovery from corrupted storage with quorum loss
+
+This example has some tricky steps highlighted in bold, where you need to be extra careful to avoid losing data.
+**Don't use Minikube, as it doesn't have full volume support.**
+
+First, [deploy the Strimzi Cluster Operator and kafka cluster](/sessions/001).
+
+When the cluster is ready, break 1 zookeeper node by writing corrupted data into ZK log.
+
+```sh
+$ CLUSTER_NAME="my-cluster"
+
+$ kubectl exec $CLUSTER_NAME-zookeeper-0 -- sh -c "echo 'test' > /var/lib/zookeeper/data/version-2/log.200000001"  \
+  && kubectl delete po $CLUSTER_NAME-zookeeper-0
+pod "my-cluster-zookeeper-0" deleted
+
+$ kubectl get po -l strimzi.io/name=$CLUSTER_NAME-zookeeper
+NAME                     READY   STATUS             RESTARTS        AGE
+my-cluster-zookeeper-0   0/1     CrashLoopBackOff   5 (2m16s ago)   16m
+my-cluster-zookeeper-1   1/1     Running            0               16m
+my-cluster-zookeeper-2   1/1     Running            0               16m
+
+$ kubectl logs $CLUSTER_NAME-zookeeper-0 | grep "Unable to load database on disk" | tail -n1
+2024-03-01 08:53:40,806 ERROR Unable to load database on disk (org.apache.zookeeper.server.quorum.QuorumPeer) [main]
+```
+
+We need to remove all data from the failed Zookeeper volumes to allow it get re-synced with the leader.
+Double check that the volume access mode is not RWOP(ReadWriteOncePod). 
+If it's the case, we can only go with offline recovery process above.
```

Comment:
I added the `RWOP` case, since for read/write mode, there are RWO(one node), RWX(many nodes), and RWOP. If it's RWOP, the only way to safe it is to delete kafka CR to save the storage.

ref: https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes

Reply:
I don't think Strimzi uses RWOP, so I would leave it out.

Reply:
OK, updated.